feeder/database/models.py

```diff
@@ -441,9 +441,6 @@ async def get(cls, pet_id: int = None, device_hid: str = None):
             logger.error("No pets found with ID: %d", pet_id)
             raise HTTPException(404, detail=f"No pet found with ID {pet_id}")
 
-        if not results:
-            raise HTTPException(404, detail="No pets found!")
```

Comment:
This API design didn't make sense. Asking for the list of pets that exist when there are no pets shouldn't result in a 404, it should return the empty list.